engine/clients/gsi/search.py

```diff
@@ -26,17 +27,24 @@ def search_one(cls, vector, meta_conditions, top) -> List[Tuple[int, float]]:
         dataset_list = cls.client.datasets_apis.controllers_dataset_controller_get_datasets_list(
             cls.client.allocation_id
             ).datasets_list
+        if DEBUG: print("GSISearcher: search_one: dataset_list=", [ d["id"] for d in dataset_list ] )
         dataset_id = dataset_list[-1]['id']
 
+        if DEBUG:
+            a = np.load(GSI_DEFAULT_QUERY_PATH)
+            print("GSISearcher: search_one: vec", a, dataset_id, a.shape, a.dtype, GSI_DEFAULT_QUERY_PATH)
 
         response = cls.client.search_apis.controllers_search_controller_search(
             SearchRequest(allocation_id=cls.client.allocation_id, dataset_id=dataset_id, 
-                          queries_file_path=GSI_DEFAULT_QUERY_PATH, topk=top, ef_search=ef),
+                          queries_file_path=GSI_DEFAULT_QUERY_PATH, topk=top), #, ef_search=ef),
```

Comment:
ef_search is None by default, when not provided in the experiment config it is still None, but it's only useful for GSI HNSW which doesn't work anyways